dev_guide/new_app.adoc

```diff
@@ -68,8 +68,11 @@ path:
 $ osc new-app <myregistry.com>/<mycompany>/<image_name>
 ----
 
-You can also create an application from a remote repository using a label. For 
-example, to create an application from the `ruby-hello-world` remote repository:
+You can also create an application from a remote repository using a label.
```

Comment:
This kind of implies the label has some part in creating the app, or that it's somehow specific to creating from a remote repo. I'd say something like "You can also apply a label when creating an application."